_includes/markdown/Markup.md

```diff
@@ -362,18 +406,29 @@ If you would like to use an inline SVG. Here are some considerations to improve
 
 Let's put it all together in an example:
 
-```
+```html
 <svg version="1.1" width="50" height="50" role="img" aria-labelledby="title-2 desc-2">
     <title id="title-2">red square</title>
     <desc id="desc-2">A plain red square defined at 50 pixels length and height.</desc>
     <rect width="50" height="50" fill="#cc0000" />
 </svg>
 ```
 
+### SVG Icons or Icon Font?
+
+Using either SVG icons or an icon font are both suitable approaches at the end of the day. Here are some considerations:
```

Comment:
@colorful-tones I don't think we're using icon font anymore, I wouldn't even mention them in here.